docs/server/v5/server/installation/configuration.md

```diff
@@ -9,7 +9,7 @@ When you don't change the configuration, EventStoreDB will use sensible defaults
 You can check what version of EventStoreDB you have installed by using the `--version` parameter in the command line. For example:
 
 ```
-$ eventstore --version
+$ eventstored --version
```

Comment:
EventStore.ClusterNode.exe for windows ?

Reply:
I second, we might need the tab switch here.